Jenkinsfile

```diff
@@ -12,11 +12,23 @@ def runSctProvisionTest(String backend){
     echo "Going to Provision test on ${backend} for Pull Request ID: ${env.CHANGE_ID}"
     checkout scm
     def distro_cmd = ""
+    def test_cmd = "run-test"
+    def test_name = "longevity_test.LongevityTest.test_custom_time"
+
     def sct_config_files = "test-cases/PR-provision-test.yaml"
     if (backend == 'gce') {
         distro_cmd = "export SCT_SCYLLA_LINUX_DISTRO='centos'"
     } else if (backend == 'docker') {
         sct_config_files = "test-cases/PR-provision-test-docker.yaml"
+    } else if (backend == 'k8s-local-kind-aws') {
+        sct_config_files = "test-cases/scylla-operator/functional.yaml"
+        test_name = "functional_tests/scylla_operator"
+        test_cmd = "run-pytest"
+        distro_cmd = """
+        export SCT_DOCKER_IMAGE='scylladb/scylla-nightly'
+        export SCT_SCYLLA_VERSION='latest'
```

Comment:
Why exactly latest Scylla must be run? Why not some stable?
We are going to test compatibility here and not Scylla itself.
So, I think we must use stable Scylla version here.
Then, the operator version is not specified here. But must be. And must also be a stable one too.

Reply:
weird, running it again, and it's passed just fine...